aodn/geonetwork-build
# added 19115-3 v1 schema plugin

@craigrose I need to add the older version of the 19115-3 standard to geonetwork-build so that we can handle harvesting from remotes that have their data in this format

To do this I basically grabbed all the files/folders from https://github.com/metadata101/iso19115-3/tree/3.5.x/src/main/plugin/iso19115-3
 and added them to the schema-plugins directory. 

To test:
- mvn clean package
- run
- confirm the schema plugin in in your local data dir: `./config/schema_plugins/iso19115-3`
  - if not you might need to clear out your local data directory then run again
- configure a harvesters
  - type: CSW
  - service url: https://ecat.ga.gov.au/geonetwork/srv/eng/csw?service=CSW&request=GetCapabilities&acceptVersions=2.0.2
  - output schema: http://standards.iso.org/iso/19115/-3/mdb/1.0
- run the harvester
- in the Editor Board
  - tick the box for the harvester under 'provided by'
  - confirm that under 'metadata standard' facet that all the records are `iso19115-3`
